src/Network/HTTPException/AcceptedException.php

```diff
@@ -26,4 +26,5 @@
 class AcceptedException extends HTTPException
 {
 	protected $code = 202;
+	var $httpdesc   = 'Accepted';
```

Comment:
Please mention the variable scope. If it isn't meant to be modified, please create it `protected` and create an accessor method in `HTTPException` like the base `Exception->getCode` and `Exception->getMessage`.

Reply:
I just have replicated what had been done at `ImATeapotException`. What do you mean with "mention the variable scope"?

Reply:
Like this:
```suggestion
	protected $httpdesc   = 'Accepted';
```
By default with `var` class members are public, any other scope needs to be explicitly mentioned.